templates/multisig/README.md

```diff
@@ -0,0 +1,129 @@
+# Aragon 0.8 Fundraising Multisig Template
+
+## Description
+
+The `fundraising-multisig` template relies on two category of actors: the board and the share holders.
+
+The board are the ones being funded by the fundraising. They are represented through a custom `BOARD` token and a dedicated voting app set to be used as a multisig.
+
+The share holders are the one contributing to the fundraising. They are represented through a `SHARE` bonded-token they can buy through the `MarketMaker` and a voting app.
+
```

Comment:
would be nice to have a simple text outline about who should be able to decide on what aspect of the DAO - fundamental power distribution: e.g. BOARD decides everything, BOARD needs to ask SHARE for certain things, SHARE can decide things on their own (they can't :D). 
Also would be nice to have some info on how to deploy the template (like in v0.7 - what method to call, what params are expected)
some roles changed from the non-functional description that was available in the audit (e.g. only board members can create vote on share, ...). this should and I think it is going to be re-audited with a focus on the template as the template is critical to the security of the system.